bin/node/runtime/src/lib.rs

```diff
@@ -549,6 +555,16 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
 	type MinerMaxWeight = MinerMaxWeight;
 	type MinerMaxLength = MinerMaxLength;
 	type MinerTxPriority = MultiPhaseUnsignedPriority;
+	type SignedMaxSubmissions = SignedMaxSubmissions;
+	type SignedRewardBase = SignedRewardBase;
+	type SignedRewardFactor = (); // no score-based reward
+	type SignedRewardMax = SignedRewardBase;
+	type SignedDepositBase = SignedDepositBase;
+	type SignedDepositByte = SignedDepositByte;
+	type SignedDepositWeight = ();
+	type SignedMaxWeight = MinerMaxWeight;
+	type SlashHandler = (); // burn slashes
```

Comment:
probably should go somewhere.

Reply:
I'm not sure what change is necessary here. If the slashes should go somewhere, then a runtime author can add whatever slash handler they want, right?. Burning it in this example node shouldn't hurt anything.

Reply:
indeed, the example node is fine. Not sure what to do for polkadot.